Docs: Add and organize panels (#51763)
grafana/grafana: docs/sources/dashboards/add-organize-panels.md

   - **Query tab -** Select your data source and enter queries here. For more information, refer to [Add a query]({{< relref "../panels/query-a-data-source/add-a-query/" >}}).
   - **Transform tab -** Apply data transformations. For more information, refer to [Transform data]({{< relref "../panels/transform-data/" >}}).
   - **Alert tab -** Write alert rules. For more information, refer to [Overview of Grafana 8 alerting]({{< relref "../alerting/" >}}).

1. Panel display options: The display options section contains tabs where you configure almost every aspect of your data visualization, including:

   - [Apply color to series and fields]({{< relref "../panels/working-with-panels/apply-color-to-series/" >}})
   - [Format a standard field]({{< relref "../panels/working-with-panels/format-standard-fields/" >}})
   - [Add a title and description to a panel]({{< relref "../panels/working-with-panels/add-title-and-description/" >}})

> Not all options are available for each visualization.

{{< figure src="/static/img/docs/panel-editor/panel-editor-8-0.png" class="docs-image--no-shadow" max-width="1500px" >}}

> ## Navigate the Grafana Inspector panel

Reviewer: ```suggestion
## Open the panel inspect drawer 
```

We never call it panel inspector in the application. In the menu it's just "Inspect"

> The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated values (CSV) file, view query requests, and export panel and data JSON.
> 
> > **Note:** Not all panel types include all tabs. For example, dashboard list panels do not have raw data to inspect, so they do not display the Stats, Data, or Query tabs.
> 
> The panel inspector consists of the following options:
> 
> 1. The panel inspector displays Inspect: <NameOfPanelBeingInspected> at the top of the pane. Click the arrow in the upper right corner to expand or reduce the pane.

Reviewer: ```suggestion
The inspect drawer helps you understand and troubleshoot your panels. You can view the raw data for any panel, export that data to a comma-separated values (CSV) file, view query requests, and export panel and data JSON.

> **Note:** Not all panel types include all tabs. For example, dashboard list panels do not have raw data to inspect, so they do not display the Stats, Data, or Query tabs.

The panel inspector consists of the following options:

1. The panel inspect drawer displays opens a drawer on the right side. Click the arrow in the upper right corner to expand or reduce the drawer pane.
```
